docs/examples/basic_conceptual_guide/Part_2-dynamic_batching/Readme.md

```diff
@@ -0,0 +1,240 @@
+```
+import torch
+from utils.model import STRModel
+
+# Create PyTorch Model Object
+model = STRModel(input_channels=1, output_channels=512, num_classes=37)
+
+# Load model weights from external file
+state = torch.load("None-ResNet-None-CTC.pth")
+state = {key.replace("module.", ""): value for key, value in state.items()}
+model.load_state_dict(state)
+
+# Create ONNX file by tracing model
+trace_input = torch.randn(1, 1, 32, 100)
+torch.onnx.export(model, trace_input, "str.onnx", verbose=True, dynamic_axes={'input.1':[0],'308':[0]})
+```
+
+### Launching the server
+
+As discussed in `Part 1`, a model repository is a filesystem based repository of models and configuration schema used by the Triton Inference Server (refer to `Part 1` for a more detailed explanation for model repositories). For this example the model repository structure would need to be set up in the following manner:
+```
+model_repository
+|   
+|-- text_recognition
+    |
+    |-- config.pbtxt
+    |-- 1
+        |
+        |-- model.onnx
+```
+This repository is a subset from the previous example. The key difference in this setup is the use of `instance_group`(s) and `dynamic_batching` in the model configuration. The additions are as follows:
+
+```
+instance_group [
+    {
+      count: 2
+      kind: KIND_GPU
+    }
+]
+dynamic_batching { }
+```
+With `instance_group` users can primarily tweak two things. First, the number of instances on a that model deployed on each GPU. The above example will deploy `2` intances of the model `per GPU`. Secondly, the target GPUs for this group can be specified with `gpus: [ <device number>, ... <device number> ]`.
+
+Adding `dynamic_batching {}` will enable the use of dynamic batches. Users can also add `preferred_batch_size` and `max_queue_delay_microseconds` in the body of dynamic batching to manage more efficient batching per their use case. Explore the [model configuration](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#model-configuration) documentation for more information
+
+As the model repository setup, the Triton Inference Server can be launched.
+```
+docker run --gpus=all -it --shm-size=256m --rm -p8000:8000 -p8001:8001 -p8002:8002 -v ${PWD}:/workspace/ -v ${PWD}/model_repository:/models nvcr.io/nvidia/tritonserver:22.08-py3 bash
+
+tritonserver --model-repository=/models
+```
+
+### Measuring Performance
+
+Having made some improvements to the model's serving capablities with enabling `dynamic batching` and the use of `multiple model instances`, the next step is to make observations these changes. To that end, the Triton Inference Server comes packaged with the [Performance Analyzer](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md) which is a tool specifically designed to measure perfomance for Triton Inference Servers. For ease of use, it is recommended that users run this inside the same container used to run client code in Part 1 of this series.
```

Comment:
Nit: the subordinate clause seems to have a mistake. Maybe "the next step is to observe these changes"?

And it might be helpful to say that the tool is designed to measure the performance of models running on Triton. Since it's not explicitly measuring the performance of the server(s).